src/rtp.c

```diff
@@ -201,13 +201,6 @@ rtp_check_packet(packet_t *packet)
         if (!(stream_is_complete(stream))) {
             stream_complete(stream, src);
             stream_set_format(stream, format);
-            // Check if an stream in the opposite direction exists
-            if (!(reverse = rtp_find_call_stream(stream->media->msg->call, stream->dst, stream->src))) {
-                reverse = stream_create(stream->media, stream->src, PACKET_RTP);
-                stream_complete(reverse, stream->dst);
-                stream_set_format(reverse, format);
-                call_add_stream(msg_get_call(stream->media->msg), reverse);
-            }
```

Comment:
Having correctly created the streams based on the SDP. There's no need for this. Which by the way makes ui_call_flow take the opposite direction for the stream given that the 'media' corresponds to the other peer.